Automattic/jetpack
# Products: Update links on Backup/Security cards to direct to checkout for real-time products

Resolves `1200412004370260-as-1201289205953284`.

#### Changes proposed in this Pull Request:

* On product pages (`/admin.php?page=jetpack#/product/*`) that feature Backup or Security, update call-to-action links so that people are sent to checkout for new real-time products instead of the older Daily products.
* Update the feature list on the Jetpack Backup card to reflect the correct amount of available backup storage (10 GB instead of unlimited).

#### Jetpack product discussion

`pbtFFM-XN-p2`

#### Does this pull request change what data or activity we track or use?

No changes 👍 

#### Testing instructions:

1.  If your Jetpack site doesn't already have a Backup subscription, go to the **Jetpack > Dashboard > At A Glance** page and click the **Upgrade** button on the **Backup** card. *(Alternatively, manually go to `/wp-admin/admin.php?page=jetpack#/product/backup` in your browser.)*
2. Verify the list of features for the Backup product are updated as shown in the right-side screenshot below (10GB of storage, not unlimited).
3. Verify the buttons for adding Backup/Security each redirect to checkout for their respective real-time version, with 10GB of storage and a yearly billing term.

#### Reference screenshot

<img height="250" alt="image" src="https://user-images.githubusercontent.com/670067/140814676-523c0f74-59be-4683-b7d3-421231ea3f2e.png"> <img height="250" alt="image" src="https://user-images.githubusercontent.com/670067/140815768-5fdcbbe2-2faa-4edc-b956-3be767f11f9c.png">
